File: COM_SIC_Procesa_Pagos/DataPowerRest/AuditoriaEWS.cs
using System;

namespace COM_SIC_Procesa_Pagos.DataPowerRest
{
	[Serializable]
	public class AuditoriaEWS
	{
		private string _USRAPP ;
		private string _IPAPLICACION  ;
		private string _APLICACION  ;
		private string _IDTRANSACCION  ;
		private string _idTransaccionNegocio  ;
		private string _userId  ;
		private string _applicationCodeWS  ;
		private string _nameRegEdit  ;

		public string USRAPP
		{
			set{_USRAPP= value;}
			get{ return _USRAPP;}
		}

		public string IPAPLICACION
		{
			set{_IPAPLICACION= value;}
			get{ return _IPAPLICACION;}
		}

		public string APLICACION
		{
			set{_APLICACION= value;}
			get{ return _APLICACION;}
		}

		public string IDTRANSACCION
		{
			set{_IDTRANSACCION= value;}
			get{ return _IDTRANSACCION;}
		}

		public string idTransaccionNegocio
		{
			set{_idTransaccionNegocio= value;}
			get{ return _idTransaccionNegocio;}
		}

		public string userId
		{
			set{_userId= value;}
			get{ return _userId;}
		}

		public string applicationCodeWS
		{
			set{_applicationCodeWS= value;}
			get{ return _applicationCodeWS;}
		}

		public string nameRegEdit
		{
			set{_nameRegEdit= value;}
			get{ return _nameRegEdit;}
		}

		public AuditoriaEWS()
		{
			
		}
	}
}

File: COM_SIC_Activaciones/AuditoriaEWS.vb
Imports System.Runtime.Serialization

'PROY-140126 
<Serializable()> _
Public Class AuditoriaEWS
    Private _USRAPP As String
    Private _IPAPLICACION As String
    Private _APLICACION As String
    Private _IDTRANSACCION As String
    Public AuditoriaEWS()'PROY-140126
    'INC000002161718 inicio
    Private _idTransaccionNegocio As String
    Private _userId As String
    Private _applicationCodeWS As String
    Private _nameRegEdit As String
    'INC000002161718 fin

    'INI: INICIATIVA-219
    Private _msgId As String
    Private _timestamp As String

    Public Property msgId() As String
        Get
            Return _msgId
        End Get
        Set(ByVal Value As String)
            _msgId = Value
        End Set
    End Property

    Public Property timestamp() As String
        Get
            Return _timestamp
        End Get
        Set(ByVal Value As String)
            _timestamp = Value
        End Set
    End Property
    'FIN: INICIATIVA-219

    Public Property USRAPP() As String
        Get
            Return _USRAPP
        End Get
        Set(ByVal Value As String)
            _USRAPP = Value
        End Set
    End Property
    Public Property IPAPLICACION() As String
        Get
            Return _IPAPLICACION
        End Get
        Set(ByVal Value As String)
            _IPAPLICACION = Value
        End Set
    End Property
    Public Property APLICACION() As String
        Get
            Return _APLICACION
        End Get
        Set(ByVal Value As String)
            _APLICACION = Value
        End Set
    End Property
    Public Property IDTRANSACCION() As String
        Get
            Return _IDTRANSACCION
        End Get
        Set(ByVal Value As String)
            _IDTRANSACCION = Value
        End Set
    End Property

    'INC000002161718 inicio
    Public Property idTransaccionNegocio() As String
        Get
            Return _idTransaccionNegocio
        End Get
        Set(ByVal Value As String)
            _idTransaccionNegocio = Value
        End Set
    End Property

    Public Property userId() As String
        Get
            Return _userId
        End Get
        Set(ByVal Value As String)
            _userId = Value
        End Set
    End Property

    Public Property applicationCodeWS() As String
        Get
            Return _applicationCodeWS
        End Get
        Set(ByVal Value As String)
            _applicationCodeWS = Value
        End Set
    End Property

    Public Property nameRegEdit() As String
        Get
            Return _nameRegEdit
        End Get
        Set(ByVal Value As String)
            _nameRegEdit = Value
        End Set
    End Property
    'INC000002161718 fin
End Class
